components/Text/Text.module.scss

```diff
@@ -1,54 +1,52 @@
-// Font Size
-.textTitle,
-.textH1,
-.textH2,
-.textH3,
-.textH4 {
-  font-weight: map-get($font-weights, bold);
-}
-
-.bodyLargeRegular,
-.bodyLargeSimeBold {
-  font-size: $font-body-large;
-}
-
-.bodySmallRegular,
-.bodySmallSemiBold {
-  font-size: $font-body-small;
-}
+.text {
+  &__title,
+  &__h1,
+  &__h2,
+  &__h3,
+  &__h4 {
+    font-weight: map-get($font-weights, bold);
+  }
+  &__title {
+    font-size: $font-title;
+  }
+  &__h1 {
+    font-size: $font-h1;
+  }
+  &__h2 {
+    font-size: $font-h2;
+  }
+  &__h3 {
+    font-size: $font-h3;
+  }
+  &__h4 {
+    font-size: $font-h4;
+  }
+  &__h5 {
+    font-weight: map-get($font-weights, regular);
+    font-size: $font-h5;
+  }
+  &__body__large__regular,
+  &__body__large__semi__bold {
+    font-size: $font-body-large;
+  }
```

Comment:
here you are only dealing with font-size, then, I guess this class can be called by --size only
```suggestion
  &--large {
    font-size: $font-body-large;
  }
```

Reply:
The problem is that we have 4 variations of the body font

![Screenshot 2022-12-19 at 16 30 26](https://user-images.githubusercontent.com/41298710/208473764-2307124a-deba-4e9b-bad1-5bfb5ab6a696.png)